src/ctapipe/instrument/optics.py

```diff
@@ -263,3 +270,141 @@ def __repr__(self):
 
     def __str__(self):
         return self.name
+
+
+class PSFModel(TelescopeComponent):
+    def __init__(self, subarray, **kwargs):
```

Comment:
not needed, since identical with baseclass.

Docstring should be at the class definition, not `__init__`.

Reply:
OK, I'll fix it.